File: 14215.py
a,b,c, = map(int, input().split())

#삼각형 조건을 만족하는지 확인 
max = max(a,b,c)
tmp = a+b+c-max

if max < tmp: #조건 충족
    print(a+b+c)
else: #조건 불충족
    max = tmp-1
    print(max+tmp)

File: 14215.c
#include <stdio.h>

int main() {

  int a,b,c;
  scanf("%d %d %d", &a, &b, &c);

  //최대값구하기
  int max = a;
  if (max < b){
    max = b;
  } 
  if (max < c){
    max = c;
  }

  int tmp = a+b+c-max;

  if (max < tmp) {
    printf("%d\n", a+b+c);
  } else {
    max = tmp - 1;
    printf("%d\n", max+tmp);
  }
  
  return 0;
}
